### ISSUE TITLE
just a blank page

### ISSUE THREAD

**Issue by hopewise** [Bodacious/blogit#72]
I have followed the instructions, but when I visit http://localhost:3000/blogs, I get a blank screen, nothing, what is expected to be seen?

**Comment by meonthewire** [comment]
+1

**Comment by tbbooher** [comment]
+1 -- you can seed it and start to see posts, but you can't edit or add them. I also added the admin engine. So far, I get:

```
Routes for Blogit::Engine:
                  GET    /posts/page/:page(.:format)            blogit/posts#index
tagged_blog_posts GET    /posts/tagged/:tag(.:format)           blogit/posts#tagged
    post_comments POST   /posts/:post_id/comments(.:format)     blogit/comments#create
     post_comment DELETE /posts/:post_id/comments/:id(.:format) blogit/comments#destroy
            posts GET    /posts(.:format)                       blogit/posts#index
                  POST   /posts(.:format)                       blogit/posts#create
         new_post GET    /posts/new(.:format)                   blogit/posts#new
        edit_post GET    /posts/:id/edit(.:format)              blogit/posts#edit
             post GET    /posts/:id(.:format)                   blogit/posts#show
                  PATCH  /posts/:id(.:format)                   blogit/posts#update
                  PUT    /posts/:id(.:format)                   blogit/posts#update
                  DELETE /posts/:id(.:format)                   blogit/posts#destroy
        blog_root GET    /                                      blogit/posts#index

Routes for Blogit::Admin::Engine:
         comments GET    /comments(.:format)                         blogit/admin/comments#index
                  POST   /comments(.:format)                         blogit/admin/comments#create
      new_comment GET    /comments/new(.:format)                     blogit/admin/comments#new
```

but any admin action results in:

```
request: http://localhost:3000/blog/posts/new
The action 'new' could not be found for Blogit::PostsController
```

due to: 

```
unless action_name = _find_action_name(@_action_name)
  raise ActionNotFound, "The action '#{action}' could not be found for #{self.class.name}"
end
```

My user model:

```
class User < ActiveRecord::Base
  ROLES = %w[admin author banned]

  blogs

  include Encryption
  has_paper_trail
...
```

**Comment by hopewise** [comment]
What I expected is to see some link, like, new post,
So, does this mean that the admin part is still not implemented?

**Comment by tbbooher** [comment]
If you download the demo app, you can see such a link. I'm trying to figure out what is missing from the instructions. However the demo app can't create a basic record.

![screen shot 2015-08-04 at 6 34 55 am](https://cloud.githubusercontent.com/assets/10830/9058909/0ead9524-3a73-11e5-83be-7b8dff779e81.png)

However, even the demo app has problems with missing gems and configuration.

**Comment by hopewise** [comment]
hmm, I will wait a few days to see if this would be fixed before trying
other gems, what I liked about this gem is that its very basic

On Tue, Aug 4, 2015 at 1:37 PM, Tim Booher notifications@github.com wrote:

> If you download the demo app, you can see such a link. I'm trying to
> figure out what is missing from the instructions. However the demo app
> can't create a basic record.
> 
> [image: screen shot 2015-08-04 at 6 34 55 am]
> https://cloud.githubusercontent.com/assets/10830/9058909/0ead9524-3a73-11e5-83be-7b8dff779e81.png
> 
> However, even the demo app has problems with missing gems and
> configuration.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/KatanaCode/blogit/issues/72#issuecomment-127560197.

## 

## 

## Kind Regards,

Samir Sabri
Software Architect& Developer
www.dcaclab.com
Jordan-Middle East

**Comment by tbbooher** [comment]
+1

**Comment by mikeover** [comment]
I don't understand this gem, you install it, follow all of the instructions, you type rake routes and you'd think you could then go to the routes specified there such as "/blog/posts/new" but nope, getting the same error as @tbbooher. No real instructions on how to set this up other than merely installing the gem and that's it. Some better documentation would be very handy...

**Comment by bertomartin** [comment]
Yup, having the same problem. I really tried to use this, but as is, it's pretty bad. Definitely not a quick and simple blog as I was hoping. Time to roll up my sleeves and build my own.

**Comment by mlensment** [comment]
+1

**Comment by myzen2** [comment]
I just ask how to generate views with this engine ?

**Comment by myzen2** [comment]
I find this : there is no action new in the blogit posts_controller I think that you should add it and you will see this page.

**Comment by waclock** [comment]
Anyone solved this?
